### DISCUSSION TITLE
Encounter

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#30] [AU Core R1 Resource Priority List - CLOSED]
Upvote for inclusion of a Resource Profile of [Encounter](http://hl7.org/fhir/R4/encounter.html) Resource in AU Core Release 1.

Comment if you think we need a different profile to AU Core Encounter, or if there is a need for an additional Encounter profile in AU Core.

AU Core Encounter profile would set minimum expectations for a Encounter resource to record, search, and fetch basic encounter information for a patient. 

AU Core Encounter should be based on the [AU Base Encounter](http://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-encounter.html) profile and identify the additional mandatory core elements, extensions, vocabularies and value sets that SHALL be present in the Encounter resource when conforming to this profile. It provides the floor for standards development for specific uses cases in an Australian context.

**Comment by brettesler-ext** [comment]
Should Encounter be a place to recode progress/visit notes?

**Reply by dt-r** [reply]
Moving this to the consensus discussion: https://github.com/hl7au/au-fhir-core/discussions/62

**Comment by brettesler-ext** [comment]
Consider pushing Encounter.reasonCode binding to FHIRWG for AU Base with a recommendation.

**Reply by dt-r** [reply]
Moving this to the consensus discussion: https://github.com/hl7au/au-fhir-core/discussions/62

**Comment by gonatf** [comment]
Why have we made period mandatory whereas US Core hasn't?  This will limit the use of core encounter when the period isn't known.

**Reply by dt-r** [reply]
Moving this to the consensus discussion: https://github.com/hl7au/au-fhir-core/discussions/62

**Comment by gonatf** [comment]
Why can't the description be captured in the narrative?

**Reply by dt-r** [reply]
Moving this to the consensus discussion: https://github.com/hl7au/au-fhir-core/discussions/62
